spec/Powertrain/TractionBattery.vspec

```diff
@@ -296,13 +326,21 @@ Charging.ChargePlugType:
 Charging.Mode:
   datatype: string
   type: actuator
-  allowed: ['MANUAL', 'TIMER', 'GRID', 'PROFILE']
-  description: Control of the charge process.
+  allowed: ['DEACTIVATED', 'AUTOMATIC', 'TRIGGERED', 'TIMER', 'PROFILE', 'EXTERNAL_ENTITY', 'MANUAL', 'GRID']
+  description: Describes how the charging process is controlled.
+               DEACTIVATED means that charging and discharging is deactivated, nothing will happen if charger is connected.
+               AUTOMATIC means charging will be initiated as soon as charger is connected.
+               TRIGGERED means charging will be initiated when triggered by user.
+               TIMER means charging is timer-based.
+               PROFILE means charging is controlled by profile downloaded to vehicle.
+               EXTERNAL_ENTITY means charging/discharging is controlled by the external entity connected to the vehicle.
+               This includes GRID-controlled charging (e.g. ISO 15118), but also other cases where vehicle is
+               connected to an arbitrary load that is powered by the vehicle.
                MANUAL means manually initiated (plug-in event, companion app, etc).
-               TIMER means timer-based.
                GRID means grid-controlled (e.g. ISO 15118).
-               PROFILE means controlled by profile download to vehicle.
-  comment:     The mechanism to provide a profile to the vehicle is currently not covered by VSS.
+  deprecation: V4.1 - MANUAL and GRID are deprecated, please use AUTOMATIC/TRIGGERED or EXTERNAL_ENITY instead.
```

Comment:
Added MANUAL+GRID plus this deprecation text. Note that we here use the `deprecation` keyword in a somewhat new meaning as the signal itself is not deprecated but rather certain parts of it. We could alternatively just mention deprecation in free text